tests/bug01665-1-nonobject.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test for bug #1665: Segfault when overriding a function object parameter + collect_params > 0
```

Comment:
Can you make sure that the test descriptions are different for each of the files? The filename can then just be `-1` (or `-2`, if you rename them).

Reply:
ah, too late, I've renamed the files : ) I've also changed the test description to put the xdebug.collect_params value (seems like a good idea?)